cmd/server-main_test.go

```diff
@@ -63,20 +63,66 @@ func TestGetListenIPs(t *testing.T) {
 	}
 }
 
+// Tests get host port.
+func TestGetHostPort(t *testing.T) {
+	testCases := []struct {
+		addr string
+		err  error
+	}{
+		// Test 1 - successful.
+		{
+			addr: ":" + getFreePort(),
+			err:  nil,
+		},
+		// Test 2 port empty.
+		{
+			addr: ":0",
+			err:  errors.New("Port cannot be empty or '0', please use `--address` to pick a specific port"),
```

Comment:
@harshavardhana should we not define these somewhere so we can just reference to them?

main.go
```
var errEmptyPortSpecified = errors.New("Port cannot be empty or '0', please use `--address` to pick a specific port")

...
return errEmptyPortSpecified
...
```

main_test.go
```
{
  addr: ":" + getFreePort(),
  err: errEmptyPortSpecified,
}
```